joinmarket/yieldgenerator.py

```diff
@@ -16,6 +16,8 @@
 
 log = get_log()
 
+MAX_MIX_DEPTH = 5
```

Comment:
Removing the configurable mixdepths is troublesome for wallets that already use more than the default of 5 mixdepths, as every time you quit the yield generator it truncates the index cache, having not read more than 5 levels into memory.

I worked around this for the time being by adjusting this value. Maybe I'll start over with a new wallet at some point.